_posts/2021-09-01-Fortran-Newsletter-September-2021.md

```diff
@@ -0,0 +1,103 @@
+
+## Fortran Standard Library
+
+Here's what's new in stdlib:
+
+TODO @milancurcic
+
+Please help improve stdlib by testing and [reviewing pull requests](https://github.com/fortran-lang/stdlib/issues?q=is%3Apr+is%3Aopen+label%3A%22reviewers+needed%22)!
+
+The candidate for file system operations to be included in stdlib is being developed by
+[@MarDiehl](https://github.com/MarDiehl) and [@arjenmarkus](https://github.com/arjenmarkus)
+in [this repository](https://github.com/MarDiehl/stdlib_os).
+Please try it out and let us know how it works, if there are any issues, or if the API can be improved.
+
+## Fortran Package Manager
+
+Here's what's new in fpm:
+
+TODO @milancurcic
+
+`fpm` is still in early development and we need as much help as we can get.
+Here's how you can help today:
+
+* Use it and let us know what you think! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm, and the [manifest reference](https://github.com/fortran-lang/fpm/blob/master/manifest-reference.md) to learn what are all the things that you can specify in the fpm.toml file.
+* Browse existing *fpm* packages on the [fortran-lang website](https://fortran-lang.org/packages/fpm)
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm and submit it to the [Registry](https://github.com/fortran-lang/fpm-registry).
+* Improve the documentation.
+
+The short-term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+## Compilers
+
+### Flang
+
+TODO @AlexisPerry
+
+### LFortran
+
+TODO @certik
+
+We are looking for new contributors, so if you are interested, please [get in
+touch with us](https://lfortran.org/)!
+
+
+## Events
+
+* We had our 15th Fortran Monthly call on August 17.
+  You can watch the recording below:
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/9goA20JAHls" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+* FortranCon 2021 will be held virtually from September 23-24, 2021.
+  Registration is free of charge and is due by September 15.
+  The second call for abstracts is due September 1.
+  For more information, visit the [FortranCon website](https://tcevents.chem.uzh.ch/event/14/).
+
+* Our Google Summer of Code program for 2021 is coming to a close.
+  Read about our students and their progress so far on Discourse: <https://fortran-lang.discourse.group/c/gsoc-2021/11>
```

Comment:
Should we link to all final GSoC reports here?

Reply:
Yes, and to their final reports too.